### DISCUSSION TITLE
Thermal Points

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#306] [Show and tell]
New in commit 68efb8b135dbd22779e3355f4a5a6baf5abbc76e any Thermal Thresholds and Limits which can be collected.

![2021-12-18-163718_724x589_scrot](https://user-images.githubusercontent.com/11563789/146646692-a346f216-9f08-440a-9583-9c45b78a7204.png)

Additional registers will be append dependending of specifications availability ...

**Comment by cyring** [comment]
Various improvements including Fahrenheit conversion and french translation of the window.

![2021-12-18-223023_724x589_scrot](https://user-images.githubusercontent.com/11563789/146655962-3c5453a2-7470-4f9a-98c3-69641d2af92b.png)

![2021-12-18-223247_724x589_scrot](https://user-images.githubusercontent.com/11563789/146656000-92aeff62-9d79-4ca0-9799-102d8d2dd825.png)

**Comment by jowa2021** [comment]
I will gather some, this is my main focus area.. I am not very familiar with French though

**Comment by cyring** [comment]
* As specified in the Intel SDM, Core and Package MSR_IA32_THERM_INTERRUPT registers are available if `CPUID( EAX=1 )` returns `EDX[ bit:22 ] == 1`

1. specs
https://github.com/cyring/CoreFreq/blob/d253207ee4fc5ce0ebe107b47a306e6b4468d95b/coretypes.h#L732

2. condition
https://github.com/cyring/CoreFreq/blob/d253207ee4fc5ce0ebe107b47a306e6b4468d95b/corefreqk.c#L7708

* In _CoreFreq_, Features window, that capability bit is shown in `ACPI`
* However, even if capable, threshold values may not be available, thus showing `[Missing]` in UI

![2021-12-19-101114_724x934_scrot](https://user-images.githubusercontent.com/11563789/146669656-aad9a818-b4f3-4286-8082-12a2b67be3bf.png)

* I don't have any Intel processor to show the evidence of those registers. Feel free to post your screenshot result.

**Reply by jowa2021** [reply]
I am having same issue.. any idea where I could look to fix this ? none of the values are editable ..... they used to be though, i remember using this tool and it working...
![image](https://user-images.githubusercontent.com/80190819/147401165-9621a7f9-7fa1-4df5-822d-b2d130edc257.png)

**Reply by cyring** [reply]
> I am having same issue.. any idea where I could look to fix this ? none of the values are editable ..... they used to be though, i remember using this tool and it working... ![image](https://user-images.githubusercontent.com/80190819/147401165-9621a7f9-7fa1-4df5-822d-b2d130edc257.png)

Which branch and commit are you using ?
I need the full screen because I don't see the processor in question .

**Comment by cyring** [comment]
* Thermal Points test from a Lucienne/5300U

![2021-12-19-110021_724x364_scrot](https://user-images.githubusercontent.com/11563789/146670977-c10722b3-fac4-42d7-8027-aae5850b44eb.png)

* Observe the `Thermal Monitor Trip` difference with the Matisse/3950X

![2021-12-19-110209_724x589_scrot](https://user-images.githubusercontent.com/11563789/146671039-0b96aeec-b14b-4bb9-9997-a6417013ed9a.png)

**Comment by cyring** [comment]
# EPYC Thermal Points

![2021-12-29-124929_802x579_scrot](https://user-images.githubusercontent.com/11563789/147659335-bc1effae-fbf3-46f0-b0fd-59ba0304034c.png)
